frontend/src/components/Dashboard.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import AppointmentCard from './AppointmentCard';
+import PyschiatristCard from './PsychiatristCard';
+import ArticleCard from './ArticleCard';
+
+const names = ["Erica Smithson", "Ronald River", "Hermes Heroic", "Pomelo Pigskin"]
+const starts = ['2023-04-10 13:30', '2023-05-10 12:30', '2023-06-15 14:15', '2023-06-20 14:15']
+const ends = ['2023-04-10 14:30', '2023-05-10 13:30', '2023-06-15 15:15', '2023-06-20 15:15']
+const dummy_descrip = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam dignissim efficitur sapien id facilisis. In imperdiet ante ac fermentum aliquet. Nam pellentesque vestibulum nisl quis facilisis. Sed eget magna auctor."
+const dummy_title = "This is the title of a mental health resource article";
```

Comment:
All this data can be represented in a dummy JSON file rather than hard coding into your file directly. Then you can use `map()` to dynamically generate components. This will simulate how you transform API responses from the backend into components on the frontend.